SMLHelper/MonoBehaviours/Fixer.cs

```diff
@@ -7,61 +7,35 @@ namespace SMLHelper.V2.MonoBehaviours
     /// <summary>
     /// This <see cref="MonoBehaviour"/> is <c>automatically</c> added to any <see cref="GameObject"/> created through <see cref="ModPrefab.GetGameObject"/>.
     /// </summary>
-    public class Fixer : MonoBehaviour, IProtoEventListener
```

Comment:
I think it's safe to remove `IProtoEventListener` here and also make `Fixer` internal, because its lifetime is just one second. But maybe I'm missing something ?

Reply:
Can you confirm that without `IProtoEventListener`, the cloned objects don't just return upon loading the save again?

Reply:
No, they shouldn't. I did a lot of tests with different mods and loading/saving (including quitting to main menu and loading again). And Fixer either removing prefab or removing itself from object, so it won't be saved anyway.
While loading, Fixers are created the same way, by ModPrefab.
I think that maybe `IProtoEventListener` was needed when Fixer didn't remove itself from object.

Reply:
Works for me